recipes/hpp-gui/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set name = "hpp-gui" %}
+{% set version = "4.10.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/humanoid-path-planner/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 677d3bd69e98c0a158a2525d36baf2c0e022968bf6e3a92f3e690d5eb0689393
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage(name|lower, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - pkg-config
+    - doxygen
+    - {{ cdt('mesa-libgl-devel') }}      # [linux]
+    - {{ cdt('mesa-dri-drivers') }}      # [linux]
+    - {{ cdt('libselinux-devel') }}      # [linux]
+    - {{ cdt('libxdamage-devel') }}      # [linux]
+    - {{ cdt('libx11-devel') }}          # [linux]
+    - {{ cdt('libxfixes') }}             # [linux]
+    - {{ cdt('libxxf86vm') }}            # [linux]
+    - {{ cdt('libxext') }}               # [linux]
+  host:
+    - qt
+    - boost
+    - boost-cpp
```

Comment:
Our `boost` package is `boost-cpp` + Boost.Python. If not, using Boost.Python you can remove `boost` from `host` and `run`.

Reply:
Thanks for the precision, will revise that